src/processes.c

```diff
@@ -1529,8 +1529,14 @@ static char *ps_get_owner(pid_t pid)
         uid = atoi (line + 5);
         getpwuid_r (uid, &passwd, passwd_buffer, sizeof(passwd_buffer),
                 &passwd_result);
-        if (passwd_result)
+        if (passwd_result) {
             result = sstrdup (passwd_result->pw_name);
+        } else {
+	    // Send the numeric uid if name is not available.
+            char uid_str[6];  // 5 digits plus string terminator.
+            snprintf(uid_str, sizeof(uid_str), "%u", uid);
```

Comment:
Better yet, since we don't care about retaining the original line:
```
char* uid_end;
uid = strtoul (line + 5, &uid_end, /* base */ 10);
...
} else {
  *uid_end = '\0';
  result = sstrdup (line + 5);
}
```